forwards_paper.tex

```diff
@@ -147,38 +151,37 @@ \section*{Introduction}
 for many biological population sizes of interest.
 
 However, it is thought that most genetic variation is selectively neutral (or nearly so).
-By definition, the neutral alleles carried by individuals in a population
+By definition, neutral alleles carried by individuals in a population
 do not affect the population process.
 For this reason, if one records the entire genealogical history of a population over the course of a simulation,
 simply laying down neutral mutations on top of that history afterwards
-is equivalent to having generated them during the simulation.
-Precisely, we would need to know the genealogical tree relating all sampled individuals
+is equivalent to having generated them during the simulation:
+it does not matter if we generate each generation's mutations during the simulation, or afterwards.
```

Comment:
trying to explain this point about "can just throw them down afterwards" more better as it's not obvious to everyone